package/MDAnalysis/lib/util.py

```diff
@@ -1704,10 +1705,10 @@ def group_consecutive_integers(arr):
 
     Examples
     >>> arr = np.array([ 2,  3,  4,  7,  8,  9, 10, 11, 15, 16])
-    >>> group_consecutive_integers(arr)
+    >>> group_same_or_consecutive_integers(arr)
     [array([2, 3, 4]), array([ 7,  8,  9, 10, 11]), array([15, 16])]
     """
-    return np.split(arr, np.where(np.ediff1d(arr)-1)[0]+1)
+    return np.split(arr, np.where(np.ediff1d(arr)-1 > 0)[0] + 1)
```

Comment:
Changed to accept same integers as well, in case the user really wants multiple atoms from the same residue.